src/Spice86.Core/Emulator/Memory/ExpandedMemoryManager.cs

```diff
@@ -0,0 +1,777 @@
+﻿namespace Spice86.Core.Emulator.Memory;
+
+using Spice86.Core.Emulator.InterruptHandlers;
+using Spice86.Core.Emulator.VM;
+
+using System;
+using System.Buffers;
+using System.Collections.Generic;
+using System.Linq;
+
+/// <summary>
+/// Provides DOS applications with EMS memory.
+/// </summary>
+public sealed class ExpandedMemoryManager : InterruptHandler {
```

Comment:
Should probably be in Spice86.Core.Emulator.InterruptHandlers.Dos namespace

Reply:
Done !